### ISSUE TITLE
"Value for sha256 cannot be None" error while running conda create

### ISSUE THREAD

**Issue by wkretzsch** [conda/conda#5278]
Hi, I got the following message while running `conda create`. Any ideas?
```
Fetching package metadata .............An unexpected error has occurred.
Please consider posting the following information to the
conda GitHub issue tracker at:

    https://github.com/conda/conda/issues



Current conda install:

               platform : linux-64
          conda version : 4.3.11
       conda is private : False
      conda-env version : 4.3.11
    conda-build version : not installed
         python version : 3.6.0.final.0
       requests version : 2.12.4
       root environment : /home/ubuntu/miniconda3  (writable)
    default environment : /home/ubuntu/miniconda3
       envs directories : /home/ubuntu/miniconda3/envs
                          /home/ubuntu/.conda/envs
          package cache : /home/ubuntu/miniconda3/pkgs
                          /home/ubuntu/.conda/pkgs
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://conda.anaconda.org/biobuilds/linux-64
                          https://conda.anaconda.org/biobuilds/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : None
           offline mode : False
             user-agent : conda/4.3.11 requests/2.12.4 CPython/3.6.0 Linux/3.13.0-117-generic debian/jessie/sid glibc/2.19
                UID:GID : 1000:1000



`$ /home/ubuntu/miniconda3/bin/conda create --quiet -y --file requirements.txt -n myproject -c bioconda -c biobuilds -c conda-forge`




    Traceback (most recent call last):
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/exceptions.py", line 616, in conda_exception_handler
        return_value = func(*args, **kwargs)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/cli/main.py", line 137, in _main
        exit_code = args.func(args, p)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/cli/main_create.py", line 68, in execute
        install(args, parser, 'create')
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/cli/install.py", line 359, in install
        execute_actions(actions, index, verbose=not context.quiet)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/plan.py", line 825, in execute_actions
        execute_instructions(plan, index, verbose)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/instructions.py", line 258, in execute_instructions
        cmd(state, arg)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/instructions.py", line 118, in UNLINKLINKTRANSACTION_CMD
        txn = UnlinkLinkTransaction.create_from_dists(index, prefix, unlink_dists, link_dists)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/core/link.py", line 123, in create_from_dists
        for dist, pkg_dir in zip(link_dists, pkg_dirs_to_link))
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/core/link.py", line 123, in <genexpr>
        for dist, pkg_dir in zip(link_dists, pkg_dirs_to_link))
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/gateways/disk/read.py", line 88, in read_package_info
        paths_data = read_paths_json(extracted_package_directory)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/gateways/disk/read.py", line 150, in read_paths_json
        paths=(PathDataV1(**f) for f in data['paths']),
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 702, in __call__
        instance = super(EntityType, cls).__call__(*args, **kwargs)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 719, in __init__
        setattr(self, key, kwargs[key])
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 424, in __set__
        instance.__dict__[self.name] = self.validate(instance, self.box(instance, val))
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 581, in box
        return self._type(v if isinstance(v, et) else et(**v) for v in val)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 581, in <genexpr>
        return self._type(v if isinstance(v, et) else et(**v) for v in val)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/gateways/disk/read.py", line 150, in <genexpr>
        paths=(PathDataV1(**f) for f in data['paths']),
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 702, in __call__
        instance = super(EntityType, cls).__call__(*args, **kwargs)
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 719, in __init__
        setattr(self, key, kwargs[key])
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 424, in __set__
        instance.__dict__[self.name] = self.validate(instance, self.box(instance, val))
      File "/home/ubuntu/miniconda3/lib/python3.6/site-packages/conda/_vendor/auxlib/entity.py", line 465, in validate
        raise ValidationError(getattr(self, 'name', 'undefined name'), val)
    conda._vendor.auxlib.exceptions.ValidationError: Value for sha256 cannot be None.
```

**Comment by wkretzsch** [comment]
PS This is the contents of my `requirements.txt` file:
```
biopython
bwa
fastx-toolkit=0.0.14
java-jdk=8.0.92=1
pandas
python=3.5.0=1
pytest
pytest-flakes
pytest-pep8
samtools=1.3.1
```

**Comment by kalefranz** [comment]
Latest version of conda is 4.3.17.  Can you confirm that this isn't already fixed?

    conda update conda

**Comment by kalefranz** [comment]
Should be fixed with bbf80adc403e74be86e15ae1305aee061b493477

**Comment by wkretzsch** [comment]
Yup, all fixed.
